cluster/gce/upgrade.sh

```diff
@@ -101,6 +102,48 @@ function upgrade-master() {
   wait-for-master
 }
 
+# TODO(mikedanese): delete when we don't support < 1.6
+function backfile-kubeletauth-certs() {
+  if [[ ! -z "${KUBEAPISERVER_CERT_BASE64:-}" && ! -z "${KUBEAPISERVER_CERT_BASE64:-}" ]]; then
+    return 0
+  fi
+
+  mkdir -p "${KUBE_TEMP}/pki"
+  echo "${CA_KEY_BASE64}" | base64 -d > "${KUBE_TEMP}/pki/ca.key"
+  echo "${CA_CERT_BASE64}" | base64 -d > "${KUBE_TEMP}/pki/ca.crt"
+  (cd "${KUBE_TEMP}/pki"
+    download-cfssl
```

Comment:
This is not a copy paste. The existing scripts split creation between easy-rsa and cfssl for some reason and require creating *everything* in one swoop incluing the CA. We probably want to just move to cfssl for everything and consolidate all this (going to work on that in a followup). This usage seems fairly straightforward though.